<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InviteDialog</class>
 <widget class="QDialog" name="InviteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gomoku Game Plugin - Invite</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Opponent:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="leJid">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select resource:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbResource">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnBlack">
       <property name="text">
        <string>Play Black</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWhite">
       <property name="text">
        <string>Play White</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbResource</tabstop>
  <tabstop>btnBlack</tabstop>
  <tabstop>btnWhite</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnBlack</sender>
   <signal>clicked()</signal>
   <receiver>InviteDialog</receiver>
   <slot>acceptBlack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWhite</sender>
   <signal>clicked()</signal>
   <receiver>InviteDialog</receiver>
   <slot>acceptWhite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>InviteDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>acceptBlack()</slot>
  <slot>acceptWhite()</slot>
 </slots>
</ui>
